pkgs/applications/networking/cluster/lens/default.nix

```
@@ -24,6 +24,9 @@ appimageTools.wrapType2 {
     ''
       mv $out/bin/${name} $out/bin/${pname}
 
+      wrapProgram $out/bin/${name} \
```

```suggestion
      wrapProgram $out/bin/${pname} \
```

The file is renamed to `${pname}` the line before this one :)

---

Done